patterns/dswp-horizontal-card-large-img-right.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Title: DSWP Horizontal Card Large Img - Right
+ * Slug: design-system-wordpress-theme/dswp-horizontal-card-large-img-right
+ * Categories: text
+ *
+ * @package Design-System-WordPress-Theme
+ */
+?>
+
+<!-- wp:group {"templateLock":false,"lock":{"move":false,"remove":false},"metadata":{"categories":["text"],"patternName":"design-system-wordpress-theme/dswp-horizontal-card","name":"DSWP Horizontal Card Large Img"},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:columns {"templateLock":false,"lock":{"move":false,"remove":false},"align":"wide","className":"dswp-horizontal-card-large-img","style":{"border":{"width":"0px","style":"none"}}} -->
+<div class="wp-block-columns alignwide dswp-horizontal-card-large-img" style="border-style:none;border-width:0px"><!-- wp:column {"verticalAlignment":"stretch","width":"60%","style":{"spacing":{"padding":{"top":"0","bottom":"0","right":"var:preset|spacing|40","left":"var:preset|spacing|40"}}},"layout":{"type":"default"}} -->
```

Comment:
The image stacks to the bottom this way for me. If you move the "dswp-image-stack-reverse" class to this element then it puts the image on top.